content/security/authentication.md

```diff
@@ -477,9 +477,10 @@ providers: [
 
 With this in place, Nest will automatically bind `AuthGuard` to all endpoints.
 
-Now we must provide a mechanism for declaring routes as public. For this, we can create a custom decorator using the `SetMetadata` decorator factory function.
+Now we must provide a mechanism for declaring routes as public. For this, we can create a custom decorator using the `SetMetadata` decorator factory function. Create a folder named `decorators` under auth module, create a file named `public.decorators.ts` and update it as shown below
```

Comment:
I don't think we should tell the user what the folder name is supposed to be called. It might seem that nestjs relies on convention over configuration, which is not true. Suggesting just the file name is fine.

Reply:
@micalevisk i have updated text